frontend/src/components/SortableTaskList.tsx

```diff
@@ -41,15 +47,21 @@ export const SortableTaskList: FC<{
   const { t } = useTranslation();
   const [search, setSearch] = useState('');
   const [searched, setSearched] = useState(tasks);
+  const [listHeight, setListHeight] = useState(0);
   const itemHeight = 52; // enough for two lines for the task name
   const itemMargin = 6;
+  const divRef = useRef<HTMLDivElement | null>(null);
 
   useEffect(() => {
     setSearched(
       tasks.filter(({ name }) => name.toLowerCase().includes(search)),
     );
   }, [search, tasks]);
 
+  useEffect(() => {
+    if (divRef.current) setListHeight(divRef.current.clientHeight);
+  }, []);
```

Comment:
This doesn't update the height if the page is resized, it only gets the initial height.

Not sure how you would listen to the changes though.